src/python/pants/backend/kotlin/lint/ktlint/rules_integration_test.py

```diff
@@ -80,7 +80,10 @@ def rule_runner() -> RuleRunner:
 """
 
 
-def run_ktlint(rule_runner: RuleRunner, targets: list[Target]) -> FmtResult:
+def run_ktlint(
```

Comment:
Were these changes to the Kotlin ktlint backend intended for this PR?

Reply:
Absolutely not. Most have sneaked in during some debugging. Sorry for missing that.